Cargo.toml

```diff
@@ -43,7 +43,8 @@ tempfile = "3"
 
 [features]
 default = ["builtins"]
-builtins = ["slug", "percent-encoding", "humansize", "chrono", "chrono-tz", "rand"]
+builtins = ["builtins_urlencode", "slug", "humansize", "chrono", "chrono-tz", "rand"]
+builtins_urlencode = ["percent-encoding"]
```

Comment:
can you just name the feature `urlencode`?

Reply:
yes, I renamed the feature to just `urlencode`